### DISCUSSION TITLE
Build time prefix

### DISCUSSION THREAD

**Discussion by thuynh** [tailwindlabs/tailwindcss#12202] [Help]
We're currently building an internal ui component library that uses tailwind. We are using without the prefix attribute inside of the tailwind.config.js but would like to prefix everything during build time. Does anyone have a solution for this?

**Comment by alex-krasikau** [comment]
Hey @thuynh! 

You can use another configuration file with the `prefix` set up for the `build` script - https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-sibling-state.

I also want to note that Tailwind CSS doesn't modify source code, and enabling `prefix` in the configuration requires using prefixes in the source files as well, e.g. `<div class="flex>` => `<div class="ui-flex">`.
